packages/web-components/src/components/content-block-media/content-block-media-content.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { css, customElement, html } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+import styles from './content-block-media.scss';
+import DDSContentGroupSimple from '../content-group-simple/content-group-simple';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Content block media.
+ *
+ * @element dds-content-block-media-content
+ */
+@customElement(`${ddsPrefix}-content-block-media-content`)
+class DDSContentBlockMediaContent extends StableSelectorMixin(DDSContentGroupSimple) {
+  //  Overrides the media position in relation to another slotted components
+  //  eslint-disable-next-line class-methods-use-this
+  protected _renderCopy() {
+    return html`
+      <slot name="media"></slot>
```

Comment:
Did you want to put `<slot name="media">` to here, or in `_renderContent()`? Note that a shadow DOM should not have two `<slot>`s with the same name.